data-prepper-plugins/peer-forwarder/README.md

```diff
@@ -42,13 +43,59 @@ A DNS server (like [dnsmasq](http://www.thekelleys.org.uk/dnsmasq/doc.html)) can
 #### With Amazon Route 53 Private Hosted Zones
 [Private hosted zones](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/hosted-zones-private.html) enable Amazon Route 53 to "respond to DNS queries for a domain and its subdomains within one or more VPCs that you create with the Amazon VPC service." Similar to the custom DNS server approach, except that Route 53 maintains the list of Data Prepper hosts. Suffers from the same drawback in that the list must be manually kept up-to-date.
 
+### <a name="AWS_CloudMap_Discovery"></a>Using AWS CloudMap
+
+An alternative to DNS discovery is [AWS CloudMap](https://docs.aws.amazon.com/cloud-map/latest/dg/what-is-cloud-map.html).
+CloudMap provides API-based service discovery as well as DNS-based service discovery.
+
+Peer forwarder can use the API-based service discovery. To support this you must have an existing
+namespace configured for API instance discovery. You can create a new one following the instructions
+provided by the [CloudMap documentation](https://docs.aws.amazon.com/cloud-map/latest/dg/working-with-namespaces.html).
+
+Your pipeline configuration needs to include:
+
+* `awsCloudMapNamespaceName` - Set to your CloudMap Namespace name
+* `awsCloudMapServiceName` - Set to the service name within your specified Namespace
+* `awsRegion` - The AWS region where your namespace exists.
```

Comment:
This is an inconsistency which I am seeing throughout the project. Since the `awsRegion` already existed, and I'm using it in my changes, I decided to use camelCase for my new properties.